doc/writing-marble-tests.md

```diff
@@ -1,186 +1,2 @@
-
-A test example with specified values:
-
-```js
-var values = {
-  a: 1,
-  b: 2,
-  c: 3,
-  d: 4,
-  x: 1 + 3, // a + c
-  y: 2 + 4, // b + d
-}
-var e1 =    hot('---a---b---|', values);
-var e2 =    hot('-----c---d---|', values);
-var expected =  '-----x---y---|';
-
-expectObservable(e1.zip(e2, function(x, y) { return x + y; }))
-  .toBe(expected, values);
-```
-
-- Use the same hash to look up all values, this ensures that multiple uses of the same character have the
-  same value.
-- Make the result values as obvious as possible as to what they represent, these are *tests* afterall, we want
-  clarity more than efficiency, so `x: 1 + 3, // a + c` is better than just `x: 4`. The former conveys *why* it's 4,
-  the latter does not.
-
-A test example with subscription assertions:
-
-```js
-var x = cold(        '--a---b---c--|');
-var xsubs =    '------^-------!';
-var y = cold(                '---d--e---f---|');
-var ysubs =    '--------------^-------------!';
-var e1 = hot(  '------x-------y------|', { x: x, y: y });
-var expected = '--------a---b----d--e---f---|';
-
-expectObservable(e1.switch()).toBe(expected);
-expectSubscriptions(x.subscriptions).toBe(xsubs);
-expectSubscriptions(y.subscriptions).toBe(ysubs);
-```
-
-- Align the start of `xsubs` and `ysubs` diagrams with `expected` diagram.
-- Notice how the `x` cold observable is unsubscribed at the same time `e1` emits `y`.
-
-In most tests it will be unnecessary to test subscription and unsubscription points, being either obvious or implied from the `expected` diagram. In those cases do not write subscription assertions. In test cases that have inner subscriptions or cold observables with multiple subscribers, these subscription assertions can be useful.
-
-## Generating PNG marble diagrams from tests
-
-Typically, each test case in Jasmine is written as `it('should do something', function () { /* ... */ })`. To mark a test case for PNG diagram generation, you must use the `asDiagram(label)` function, like this:
-
-<!-- skip-example -->
-```js
-it.asDiagram(operatorLabel)('should do something', function () {
-  // ...
-});
-```
-
-For instance, with `zip`, we would write
-
-```js
-it.asDiagram('zip')('should zip by concatenating', function () {
-  var e1 =    hot('---a---b---|');
-  var e2 =    hot('-----c---d---|');
-  var expected =  '-----x---y---|';
-  var values = { x: 'ac', y: 'bd' };
-
-  var result = e1.zip(e2, function(x, y) { return String(x) + String(y); });
-
-  expectObservable(result).toBe(expected, values);
-});
-```
-
-Then, when running `npm run tests2png`, this test case will be parsed and a PNG file `zip.png` (filename determined by `${operatorLabel}.png`) will be created in the `img/` folder.
+- [USERS: Writing marble diagram tests for your application](./marble-testing.md).
+- [INTERNAL MAINTAINERS ONLY: Writing marble diagram tests for your the RxJS internals](./internal-marble-tests.md).
```

Comment:
I imagine even though this doc was written with the intended audience being maintainers of rxjs itself, some people have linked to this doc so to aid in their search I added this links.